command/forceleave/forceleave_test.go

```diff
@@ -56,6 +56,31 @@ func TestForceLeaveCommand(t *testing.T) {
 	})
 }
 
+func TestForceLeaveCommand_NoNodeWithName(t *testing.T) {
+	t.Parallel()
+	a1 := agent.NewTestAgent(t, t.Name(), ``)
+	a2 := agent.NewTestAgent(t, t.Name(), ``)
```

Comment:
What would happen if we only spin up one test agent and try to force leave a node that doesn't exist?

I'm wondering what benefit we get from spinning up two agents for this test.

I usually aim to minimize the number of test agents that we run in parallel, to reduce resource consumption in the CI containers. Let me know what you think.

Reply:
that makes total sense. i honestly just copy-pasted from the test above it; i'll remove the second agent. thanks!